packages/components/psammead-embed-error/src/index.jsx

```diff
@@ -1,73 +1,93 @@
 import React from 'react';
-import { string, bool } from 'prop-types';
+import { string, bool, shape } from 'prop-types';
 import styled from 'styled-components';
-import { C_CHALK, C_EBON } from '@bbc/psammead-styles/colours';
 import {
-  GEL_SPACING_TRPL,
-  GEL_SPACING_HLF,
-} from '@bbc/gel-foundations/spacings';
-import { getSansRegular } from '@bbc/psammead-styles/font-styles';
+  C_LUNAR,
+  C_SHADOW,
+  C_EBON,
+  C_METAL,
+} from '@bbc/psammead-styles/colours';
+import { GEL_SPACING_TRPL, GEL_SPACING } from '@bbc/gel-foundations/spacings';
+import { getSansRegular, getSansBold } from '@bbc/psammead-styles/font-styles';
 import { GEL_BODY_COPY } from '@bbc/gel-foundations/typography';
-import { BBC_BLOCKS, coreIcons } from '@bbc/psammead-assets/svgs';
+import { BBC_BLOCKS } from '@bbc/psammead-assets/svgs';
 
 const GOLDEN_RATIO_PERCENT = '38.2%';
-const GEL_SPACING_TRPL_MINUS_HLF = '1.25rem';
-const GEL_SPACING_QRT = '0.125rem';
+const BBC_BLOCKS_WIDTH = '10rem';
+const FAUX_BBC_BLOCKS_SPACE = '6rem';
+
+const FILL_VIEWPORT_STYLES = `
+  background-position: center center;
+  background-size: ${GOLDEN_RATIO_PERCENT};
+  height: 100vh;
+`;
 
 const StyledEmbedError = styled.div`
   ${({ service }) => getSansRegular(service)}
   ${GEL_BODY_COPY};
-  background-color: ${C_CHALK};
+  background-color: ${C_LUNAR};
   background-image: url(data:image/svg+xml;base64,${BBC_BLOCKS});
-  background-position: center center;
+  background-position: center ${GEL_SPACING_TRPL};
   background-repeat: no-repeat;
-  background-size: ${GOLDEN_RATIO_PERCENT};
-  color: ${C_EBON};
+  background-size: ${BBC_BLOCKS_WIDTH};
+  color: ${C_SHADOW};
   display: flex;
   flex-direction: column;
-  height: ${({ fillViewport }) => (fillViewport ? '100vh' : '100%')};
   justify-content: flex-end;
+  padding-top: ${FAUX_BBC_BLOCKS_SPACE};
+  padding-bottom: ${GEL_SPACING_TRPL};
+  ${({ fillViewport }) => fillViewport && FILL_VIEWPORT_STYLES}
 `;
 
 const StyledErrorMessage = styled.div`
-  display: flex;
-  margin: ${GEL_SPACING_TRPL} ${GEL_SPACING_TRPL_MINUS_HLF};
+  margin: 0 ${GEL_SPACING_TRPL};
 
   strong {
+    display: block;
     font-weight: normal;
-    margin: 0 ${GEL_SPACING_HLF};
   }
-`;
 
-const IconWrapper = styled.span`
-  position: relative;
-  top: -${GEL_SPACING_QRT};
+  a {
+    ${({ service }) => getSansBold(service)}
+    color: ${C_EBON};
+    display: block;
+    margin-top: ${GEL_SPACING};
+    text-decoration: none;
+
+    &:visited {
+      color: ${C_METAL};
+    }
 
-  > svg {
-    fill: currentColor;
-    height: ${GEL_SPACING_TRPL};
-    width: ${GEL_SPACING_TRPL};
+    &:hover,
+    &:focus {
+      text-decoration: underline;
+    }
   }
 `;
 
-const EmbedError = ({ service, message, fillViewport }) => (
+const EmbedError = ({ service, message, fillViewport, link }) => (
```

Comment:
```suggestion
const EmbedError = ({
    service,
    message,
    fillViewport,
    link: { text, href  } }) => (
```

Reply:
It would be nice if we could do this, but because `link` is an optional prop, destructuring would result in an error if it's not defined.